packages/orchestration/src/orchestration-response.ts

```diff
@@ -51,6 +54,48 @@ export class OrchestrationResponse {
     return choice?.message?.content;
   }
 
+  /**
+   * Messages that can be used for subsequent prompts as message history.
+   * @param choiceIndex - The index of the choice to parse.
+   * @returns A list of all messages.
+   */
+  getAllMessages(choiceIndex = 0): ChatMessages {
```

Comment:
[req] Could you please add an additional test in `orchestration-client.test.ts` with the a sample response json file with a `multiChatMessage` response?

I wasn't able to understand the code without looking at a sample response from the Java repo, so I think this would be beneficial for us too.